src/Sentry.Bindings.Android/sentry-proguard.cfg

```
@@ -0,0 +1,5 @@
+# Because we use string-based lookups we need to keep names on selected sentry-java classes
+-keep class io.sentry.android.core.SentryAndroid { *; }
```

Don't we need to keep also:

```suggestion
-keep class io.sentry.protocol.**
-keep class io.sentry.Sentry { *; }
```